shell/prompt.go

```diff
@@ -11,10 +11,16 @@ import (
 
 // Prompt the user for text in the CLI. Returns the text entered by the user.
 func PromptUserForInput(prompt string, terragruntOptions *options.TerragruntOptions) (string, error) {
-	terragruntOptions.Logger.Print(prompt)
-
+	n, err := terragruntOptions.ErrWriter.Write([]byte(prompt))
```

Comment:
NIT: could you add a comment that we should NOT be using a logger here, as this prompt should always be visible, regardless of log level? And perhaps cross-link to the relevant issue?